spyderlib/app/spyder.py

```diff
@@ -439,6 +439,9 @@ def signal_handler(signum, frame=None):
                                                sys.version_info[1])
         if DEBUG:
             title += " [DEBUG MODE %d]" % DEBUG
+        if not options.window_title is None:

```

Comment:
`if options.window_title is not None:` is more commonly used in python, because it looks like an english sentence.